charts/teuto-portal/templates/secret.yaml

```diff
@@ -0,0 +1,12 @@
+{{- if  not .Values.worker.existingSecret }}
```

Comment:
I'm thinking, are we ever going to not use the existing DB? I assume not 🤔

So I guess we can just use the existing secret and require it, just like the kubeconfig

Reply:
removed